wrap/wrap_test.go

```diff
@@ -0,0 +1,99 @@
+package wrap
+
+import (
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"testing"
+
+	mackerel "github.com/mackerelio/mackerel-client-go"
+	cli "gopkg.in/urfave/cli.v1"
+)
+
+func newWrapContext(args []string) *cli.Context {
+	app := cli.NewApp()
+	parentFs := flag.NewFlagSet("mockmkr", flag.ContinueOnError)
+	for _, f := range []cli.Flag{
+		cli.StringFlag{Name: "conf"}, cli.StringFlag{Name: "apibase"},
+	} {
+		f.Apply(parentFs)
+	}
+	parentFs.Parse(args)
+	for i, v := range parentFs.Args() {
+		if v == "wrap" {
+			args = parentFs.Args()[i+1:]
+			break
+		}
+	}
+	parentCtx := cli.NewContext(app, parentFs, nil)
+
+	fs := flag.NewFlagSet("mockwrap", flag.ContinueOnError)
+	for _, f := range Command.Flags {
+		f.Apply(fs)
+	}
+	fs.Parse(args)
+	return cli.NewContext(app, fs, parentCtx)
+}
+
+func TestCommand_Action(t *testing.T) {
+	ts := httptest.NewServer(http.HandlerFunc(func(res http.ResponseWriter, req *http.Request) {
+		reqPath := "/api/v0/monitoring/checks/report"
+		if req.URL.Path != reqPath {
+			t.Errorf("request URL should be %s but: %s", reqPath, req.URL.Path)
+		}
+
+		body, _ := ioutil.ReadAll(req.Body)
+		type vr struct {
```

Comment:
What does vr stand for? V... Result?

Reply:
enhanced and refined testing at e3ba37cb20774 and 5d5732c7c6.